internal/pkg/git/git_test.go

```diff
@@ -29,6 +31,13 @@ func TestGit_Checkout(t *testing.T) {
 	t.Parallel()
 	logger := log.NewDebugLogger()
 
+	// Copy the git repository to temp
+	tmpDir := t.TempDir()
+	assert.NoError(t, aferofs.CopyFs2Fs(nil, filepath.Join("test", "repository"), nil, tmpDir))
+	assert.NoError(t, os.Rename(filepath.Join(tmpDir, ".gittest"), filepath.Join(tmpDir, ".git")))
+	gitRepo := fmt.Sprintf("file://%s", tmpDir)
+
+	// Ctx
```

Comment:
Niektore testy sa spustali oproti `keboola-as-code-templates`, tam sme mergli uz produkcne sablony, ... a na Windows bol problem, ze niektore paths su dlhsie ako `256` znakov, v GHA je nejaka blba implementacia `gitu`, da sa to vyriesit, ale jednoduchsie je to testovat oproti `file://` repu.